mcfly.bash

```diff
@@ -63,8 +63,8 @@ PROMPT_COMMAND="mcfly_prompt_command;$PROMPT_COMMAND"
 #      render the search UI pre-filled with it.
 if [[ $- =~ .*i.* ]]; then
   if set -o | grep "vi " | grep -q on; then
-    bind "'\C-r': '\e0i#mcfly: \e\C-j mcfly search\C-j'"
+    bind "'\C-r': '\e0i#mcfly: \e\C-j ${MCFLY_PATH} search\C-j'"
```

Comment:
Let's leave this as just `mcfly` for now for readability in the user's shell. I'm okay if we rely on the default being bash for now.